src/NUnitFramework/framework/Attributes/TimeoutAttribute.cs

```diff
@@ -30,10 +30,10 @@
 namespace NUnit.Framework
 {
     /// <summary>
-    /// Used on a method, marks the test with a timeout value in milliseconds. 
-    /// The test will be run in a separate thread and is cancelled if the timeout 
-    /// is exceeded. Used on a class or assembly, sets the default timeout 
-    /// for all contained test methods.
+    /// Applies a timeout to a test. When used on a method it marks the test 
+    /// with a timeout value in milliseconds. The test is cancelled if the 
+    /// timeout is exceeded. When used on a class or assembly it sets the 
+    /// default timeout for all contained test methods.
```

Comment:
I see how you've tried to reword here - the original in this case was ambiguous, I think. The timeout is always in milliseconds, whether the attribute is on a method/class/assembly.